app/views/users/index.html.erb

```diff
@@ -0,0 +1,25 @@
+<%= render 'shared/flash_messages' %>
+
+<h1>Listing Users</h1>
+
+<table>
+  <thead>
+    <tr>
+      <th colspan="3"></th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <% @users.each do |user| %>
+      <tr>
+        <td><%= user.email %>
+        <td><%= link_to t('edit'), edit_user_path(user) %></td>
+        <td><%= link_to t('delete'), user, method: :delete, onclick: "return confirm('#{t('are_you_sure')}')}" %></td>
```

Comment:
I'd move the actions to a single `<td>` cell and right align it so it doesn't appear in the middle of page